add suse sls file for repo.saltstack.com

Ch3LL opened the pull request.

Ch3LL committed:
add suse sls file for repo.saltstack.com

Ch3LL commented:
@jfindlay can you please review when you get a sec

jfindlay commented on test_install/saltstack/suse.sls:
```diff
@@ -0,0 +1,71 @@
+{# Import global parameters that source from grains and pillars #}
+{% import 'params.jinja' as params %}
+
+{% set salt_version = salt['pillar.get']('salt_version', '') %}
```
@Ch3LL, this should already be defined in `params.jinja`.

jfindlay commented on test_install/saltstack/suse.sls:
```diff
@@ -0,0 +1,71 @@
+{% set salt_version = salt['pillar.get']('salt_version', '') %}
+
+{# Determine Suse_Version #}
+{% set suse_version = params.os + params.os_release %}
```
Could you get this with `grains.get('osfullname')`?

Ch3LL committed:
change suse_versiont to codename and remove salt_versions

Ch3LL commented:
@jfindlay thanks for the review. I changed suse_version to oscodename, since osfullename did not provide as much information as needed. I also removed salt_version. Once again thank you!
